frontend/util/federalistApi.js

```diff
@@ -62,6 +62,18 @@ export default {
     });
   },
 
+  resendInviteToOrganization(id, user, uaaEmail, roleId) {
```

Comment:
You can remove this method and call the above method `inviteToOrganization` with data in this "shape"

Reply:
I can! And I have.